extension/test/unit/config/config.spec.js

```diff
@@ -9,6 +9,9 @@ describe('::config::', () => {
           clientSecret: 'clientSecret',
           apiUrl: 'host',
           authUrl: 'authUrl',
+          isAuthEnabled: false,
```

Comment:
If it's false, then you don't have to provide username and password?

Reply:
it is obsoleted